src/text/3-applications-subsections/3D-analysis.tex

```diff
@@ -1,12 +1,26 @@
 \subsection{3D Analysis}
 \label{ssec:3d-analysis}
-
 \todo{Laura}
+The 1D analysis approach is a powerful tool to measure the spectrum of an isolated source. However, more complicated situations, such as field of views containing several overlapping sources, require a more careful treatment. Because it is based on integration over a region, the 1D approach can't disentangle the contribution of each source to the total flux in the selected region. Similarly, sources with extended or complex morphology can result in the measured flux being underestimated, and heavily dependent on the choice of extraction region.
+
+For such situations a more complex approach is needed, the so-called 3D analysis. The three relevant dimensions are the two spatial angular coordinates and an energy axis. In this framework, a combined spatial and spectral model (that is, a SkyModel, see Section~\ref{ssec:gammapy-modeling}) is fitted to the sky-maps that were previously derived from the data and bundled into a MapDataset (see Sections~\ref{ssec:gammapy-makers,ssec:gammapy-datasets}).
+
+Starting from the \irfs corresponding to the same three simulated \cta observations used in Section~\ref{ssec:1d-analysis}, we can create a MapDataset via the MapDatasetMaker. However, we will not use the simulated event lists provided by \cta but instead, use the method MapDataset.fake() to simulate measured counts from the combination of several SkyModel instances. In particular we simulate:
+\begin{enumerate}
+	\item A point source located at (l=0\textdegree, b=0\textdegree) with a power-law spectral shape.
+	\item An extended source with Gaussian morphology located at (l=0.4\textdegree, b=0.15\textdegree) with $\sigma$=0.2\textdegree and a log-parabola spectral shape.
+	\item A large shell-like structure centered around (l=0.06\textdegree, b=0.6\textdegree) with a radius and width of 0.6\textdegree and 0.3\textdegree respectively and a power-law spectral shape.
+\end{enumerate}
+The position and sizes of the sources have been selected so that their contributions overlap. This can be clearly seen in the significance map shown in the left panel of Figure \todo{X}. This map was produced with the ExcessMapEstimator (see Section~\ref{ssec:gammapy-estimators}) with a correlation radius of 0.1\textdegree.
+
+We can now fit the same model shapes to the simulated data and retrieve the best-fit parameters, which are in all cases well within 1$\sigma$ of the simulated parameters. We can now compute the residual significance map after removing the contribution from each model. This is done again via the ExcessMapEstimator. As can be seen in Figure \todo{X}, there are no regions above or below 5$\sigma$.
```

Comment:
Do you want to talk of agreement with the simulated parameters?

Reply:
I removed that sentence